esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1196,8 +1197,7 @@ void Esdl2Struct::addChildren(Esdl2Transformer *xformer, IEsdlDefObjectIterator
         {
             m_children.append(*obj);
             m_child_map.setValue(obj->queryName(), obj);
-            if (obj->queryEclName())
```

Comment:
This change (these two lines) is more than just a performance change - it will alter the behaviour.